p4-16/spec/P4-16-spec.mdk

```diff
@@ -6885,6 +6908,41 @@ invokes action `a`. Once the program is loaded, these entries are
 installed in the table in the order they are enumerated in the
 program.
 
+##### Entry priorities { #sec-entry-priorities }
+
+If a table has fields where their `match_kind`s are all `exact` or
+`lpm`, there is no reason to assign numeric priorities to its entries.
+If they are all `exact`, duplicate keys are not allowed, and thus
+every lookup key can match at most one entry, so there is no need for
+a tiebreaker.  If there is an `lpm` field, the priority of the entry
+corresponds to the length of the prefix, i.e. if a lookup key matches
+multiple prefixes, the longest prefix is always the winner.
+
+For tables with other `match_kind` values, e.g. at least one `ternary`
```

Comment:
How do you specify the initial priorities?

Reply:
To be described in the part of the PR that I have not yet written.  Probably another week or two.

Reply:
@mbudiu-vmw I added another commit to this PR on 2022-Nov-15 with a description of how priorities are determined.